.circleci/build_image.sh

```diff
@@ -1,4 +1,7 @@
 #!/bin/sh
+
+# FIXME(Kreyren): Oh gawd who wrote this, Abstract required for proper implementation
```

Comment:
Also i need abstract for the CircleCI to fix code quality and hopefully improve the runtime.